src/cryptography/hazmat/backends/openssl/backend.py

```diff
@@ -56,6 +57,91 @@
                                        ["code", "lib", "func", "reason"])
 
 
+def _make_asn1_int(backend, x):
```

Comment:
This function is probably simpler (and supports values > 64-bit) with the use of `BN_to_ASN1_INTEGER`